source/testcommon/testcommon/dataframes/read_csv.py

```diff
@@ -3,7 +3,11 @@
 
 
 def read_csv(
-    spark: SparkSession, path: str, schema: T.StructType, sep: str = ";"
+    spark: SparkSession,
+    path: str,
+    schema: T.StructType,
+    sep: str = ";",
+    ignored_value="[IGNORED]",
```

Comment:
This will correspond with existing test files in wholesale and measurements
```suggestion
    ignored_value="IGNORED",
```

Reply:
I discussed this with Alex, and the problem with `IGNORED` is that it also match sentences including `IGNORED`. Not that this is likely, but `[IGNORED]` is much less likely.

Reply:
In wholesale the keyword IGNORED is not used anymore.